### PR TITLE
Update db packages

### PR DESCRIPTION
**Pull request by roydahan**
- Fix issues with update_db_packages
- Support root_disk_size as a parameter for db node
- Increasing default root disk size for db node to 15GB

### CODE REVIEW

File: sdcm/tester.py [hunk 1]
```diff
@@ -474,6 +474,20 @@ def get_cluster_aws(self, loader_info, db_info, monitor_info):
                 self.fail('Unsupported parameter type: {}'.format(type(n_db_nodes)))
         if db_info['type'] is None:
             db_info['type'] = self.params.get('instance_type_db')
+        if db_info['disk_size'] is None:
+            db_info['disk_size'] = self.params.get('aws_root_disk_size_db', default=None)
+        if db_info['device_mappings'] is None:
+            if db_info['disk_size']:
+                db_info['device_mappings'] = [{
+                    "DeviceName": self.params.get("aws_root_disk_name_db", default="/dev/sda1"),
```
Comment (fruch) [inline sdcm/tester.py]: you didn't added this option to the sct_config

File: sdcm/cluster.py [hunk 2]
```diff
@@ -2152,14 +2152,11 @@ def _update_db_packages(self, new_scylla_bin, node_list):
 
         def update_scylla_packages(node, queue):
             node.log.info('Updating DB packages')
+            node.remoter.run('mkdir /tmp/scylla')
```
Comment (bentsi) [inline sdcm/cluster.py]: better `mkdir -p`

File: sdcm/tester.py (hunk 1 above)
Comment (bentsi) [reply sdcm/tester.py]: please add `aws_root_disk_name_db` to the `sct_config`

File: sdcm/cluster.py (hunk 2 above)
Comment (roydahan) [reply sdcm/cluster.py]: better that I wouldn't create it at all :)
I don't know how it worked before...

regarding -p, I think not having /tmp is something we would like to bypass.

File: sdcm/tester.py (hunk 1 above)
Comment (roydahan) [reply sdcm/tester.py]: You are right, I missed it.

Comment (roydahan) [reply sdcm/tester.py]: Fixed.

File: sdcm/cluster.py [hunk 3]
```diff
@@ -2152,14 +2152,11 @@ def _update_db_packages(self, new_scylla_bin, node_list):
...
             node.remoter.send_files(new_scylla_bin, '/tmp/scylla', verbose=True)
             node.remoter.run('sudo yum update -y --skip-broken')
-            node.remoter.run('sudo yum install python34-PyYAML -y')
```
Comment (fruch) [inline sdcm/cluster.py]: we are sure all older versions doesn't need this anymore ? i.e. I think I've added this for rolling upgrades to work

Comment (roydahan) [reply sdcm/cluster.py]: We have it also as part of the upgrade_test.py where it's called upgrade_node_packages.
I assume it's not needed at least where I changed it.

File: sdcm/cluster.py (hunk 2 above)
Comment (bentsi) [reply sdcm/cluster.py]: ok, looked in the code, we are not running `update_db_packages` in "reuse" mode. So it won't fail

**Pull request merged**